mathesar/tests/api/test_dependents_api.py

```diff
@@ -85,3 +85,19 @@ def test_column_dependents(library_ma_tables, client):
     checkouts_patrons_fk_id = [c['id'] for c in checkouts_constraints if c['name'] == 'Checkouts_Patron id_fkey']
 
     assert sorted(patrons_id_dependents_ids) == sorted([checkouts.id] + patrons_pk_id + checkouts_patrons_fk_id)
+
+
+def test_dependents_filters(library_ma_tables, client):
+    publishers_id = library_ma_tables['Publishers'].id
+    exclude_types = ['table constraint']
+    filter_param = "".join([f'exclude={type}&' for type in exclude_types])
+
+    publishers_dependents_graph = client.get(f'/api/db/v0/tables/{publishers_id}/dependents/?{filter_param}').json()
```

Comment:
`client` can accept query parameters using the data parameter. This would automatically compile the URL for us. I fixed it with the  commit 46798924e070da4a2dc2abea0d46e7be3cc2f14c